pkg/datatype/floats/slice.go

```diff
@@ -36,6 +36,18 @@ func (s Slice) Max() float64 {
 	return floats.Max(s)
 }
 
+func (s Slice) MaxIndex() int {
```

Comment:
Call it `IndexOfMaxValue` or `IndexOfMax` ?

Reply:
will do